src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2851,10 +2851,10 @@ private static bool IsAvailableFarEastCodePage(uint codePage)
         private static bool IsAnyDBCSCharSet(uint charSet)
         {
             const uint SHIFTJIS_CHARSET = 128;
-            const uint HANGEUL_CHARSET = 129;
+            const uint HANGUL_CHARSET = 129;
```

Comment:
I can drop this change if you like.

As someone who has spent over a decade working on Web Browsers, a significant amount of time dealing w/ character sets, Hangeul is definitely not the primary spelling (13million vs 1/2 million). It's a transliteration, so it isn't uncommon for there to be multiple transliterations.

Notably, the url you provided as a reference uses the `e`-less version in its enum.